cmd/minishift/cmd/start_test.go

```diff
@@ -512,7 +512,7 @@ func setUp(t *testing.T) {
 	viper.Set("host-volumes-dir", hostVolumesDirectory)
 	viper.Set("host-pv-dir", hostPvDirectory)
 
-	provision.SetDetector(&tests.MockDetector{&tests.MockProvisioner{Provisioned: true}})
+	provision.SetDetector(&tests.MockDetector{Provisioner: &tests.MockProvisioner{Provisioned: true}})
```

Comment:
This changed happen to make `go vet` happy.

Reply:
Why would that only happen now? Are you running a different version of Go?

Reply:
@hferentschik I am using 1.8.3 only.

```
$ go version
go version go1.8.3 linux/amd64
```

Reply:
So why does that not appear until now? AFAICT the vetting target is enabled for a while and you are effectively changing some unrelated code.